pay-api/src/pay_api/utils/util.py

```diff
@@ -81,3 +82,14 @@ def get_first_and_last_dates_of_month(month: int, year: int):
     start_date = datetime.now().replace(day=1, year=year, month=month)
     end_date = start_date.replace(day=calendar.monthrange(year=year, month=month)[1])
     return start_date, end_date
+
+
+def parse_url_params(url_params: str) -> Dict:
+    """Parse URL params and return dict of parsed url params."""
```

Comment:
in my test , i am using  url_param_dict = dict(urllib.parse.parse_qsl(urllib.parse.urlsplit(payment_response_url).query)).... works ok so far